### DISCUSSION TITLE
How can I  use the result of Test metric?

### DISCUSSION THREAD

**Discussion by DeepSpace98** [open-edge-platform/anomalib#915] [Q&A]
Test metric             DataLoader 0
      image_AUROC                  1.0
      image_F1Score                 1.0
the results of trainer.test‘s execution like this, but now I want to use the image_AUROC elsewhere.How to call the image_AUROC result of the test set conveniently？

**Comment by djdameln** [comment]
Hi, `trainer.test` returns a dictionary which contains the metric scores. So you can just assign the outputs to a new variable, from which you can then obtain the desired metrics:
```python
metrics = trainer.test(model=model, datamodule=datamodule)
image_auroc = metrics["image_AUROC"]
```
